nebula_ros/include/nebula_ros/velodyne/velodyne_decoder_ros_wrapper.hpp

```diff
@@ -15,6 +15,9 @@
 #include <velodyne_msgs/msg/velodyne_packet.hpp>
 #include <velodyne_msgs/msg/velodyne_scan.hpp>
 
+#include "boost/property_tree/json_parser.hpp"
+#include "boost/property_tree/ptree.hpp"
```

Comment:
```suggestion
#include <nlohmann/json.hpp>
```

Let's use this instead, boost is too heavy for this task.

`<depend>nlohmann-json-dev</depend>` can be added to the `package.xml` file and all should work fine.

And the rest of the code will be much cleaner and also will be error checked:

```cpp
// Assuming `nlohmann::json` namespace is available as `json`
void parseInvalidRegions(const std::string& regions_json) {
    // Parse the JSON string
    auto invalid_regions_json = json::parse(regions_json);

    // Clear existing regions
    sensor_configuration_.invalid_regions.clear();

    // Iterate over the parsed JSON array
    for (const auto& region : invalid_regions_json) {
        // Extract the ring number, start angle, and end angle from each sub-array
        int ring_number = region[0];
        uint16_t start = region[1];
        uint16_t end = region[2];

        // Add the InvalidRegion to the map under the corresponding ring number
        sensor_configuration_.invalid_regions[ring_number].emplace_back(InvalidRegion{start, end});
    }
}
```